package.json

```diff
@@ -15,6 +15,8 @@
     "docs:generate:api": "bun scripts/render-api-docs.ts",
     "foundryup": "curl -L https://foundry.paradigm.xyz | bash && bash ~/.foundry/bin/foundryup",
     "gas-report": "pnpm run --recursive --parallel gas-report",
+    "install:ci": "pnpm install --ignore-scripts",
+    "install:ci:frozen": "pnpm install --ignore-scripts --frozen-lockfile",
```

Comment:
my preference would be to just add `--ignore-scripts` flag to the CI command and not add new npm scripts